src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int color = new Random().nextInt(Color.values().length);
```

Comment:
Magic number detected. It's better to declare the length of the Color enum as a constant field. For example, `private static final int COLOR_COUNT = Color.values().length;` and then use this constant in `nextInt(COLOR_COUNT)`.